util/time.go

```diff
@@ -36,15 +293,21 @@ func FormatDate(tsMillis uint64) string {
 
 // Returns the current Unix timestamp in milliseconds.
 func CurrentTimeMillis() uint64 {
-	// Read from cache first.
-	tickerNow := CurrentTimeMillsWithTicker()
-	if tickerNow > uint64(0) {
-		return tickerNow
-	}
-	return uint64(time.Now().UnixNano()) / UnixTimeUnitOffset
+	return CurrentClock().CurrentTimeMillis()
 }
 
 // Returns the current Unix timestamp in nanoseconds.
 func CurrentTimeNano() uint64 {
-	return uint64(time.Now().UnixNano())
+	return CurrentClock().CurrentTimeNano()
+}
+
+// Now returns the current local time.
+func Now() time.Time {
```

Comment:
Maybe we need a benchmark to test the performance diff between `Now()` and `CurrentTimeMillsWithTicker`

Reply:
Added. 
```
BenchmarkCurrentTimeInMs/CurrentTimeMillis-4            230781794                5.26 ns/op
BenchmarkCurrentTimeInMs/CurrentTimeMillisDirect-4      519137175                2.32 ns/op
BenchmarkNow/util.Now-4                                 14527578                83.3 ns/op             
BenchmarkNow/time.Now-4                                 14849454                77.0 ns/op             
```